### ISSUE TITLE
/Users/xxxx/Library/Application Support/cloud-code/bin/versions/853df33c24eceffd71e91412744d5f0a09ce8bdf2458d174709ec59c159e0801/skaffold is out of date

### ISSUE THREAD

**Issue by awadodeh** [GoogleCloudPlatform/cloud-code-intellij#2720]
I have the latest 

- Cloud Code for IntelliJ version:
- OS: macOS Catalina

**What did you do?**
I am trying to debug my local kubernetes using Cloud Code extension. But the debug session is not getting triggered.

**What did you expect to see?**
The session should be triggered after successful start of the application.

**What did you see instead?**
the debug session is not getting triggered.

After checking the Skaffold version integrated with cloud code I see its using version `v1.3.1`, However, the latest version as of now is `v1.5.1`. I am thinking this might be the reason for issue, please advice?

**Comment by etanshaul** [comment]
Hi @awadodeh . The bundled skaffold version will be updated on the next release. If you wanted to use an up-to-date skaffold version earlier, you can point to your system-installed version by going to Settings > Cloud Code > Kubernetes and selecting you system version.

However, debugging should not be affected by this. What kind of app are you using (language, single or multi-service ?) If it is a Python app, we do not yet support debugging (this will be coming soon).

**Comment by awadodeh** [comment]
Hi @etanshaul!
I am developing a multi-service using Java Spring Boot. I found out the root cause of the issue, which was that I am missing to add the `jib` builder to my artifact definitions in `skaffold.yaml`
i.e 
```
build:
  artifacts:
    - image: xxx/api-gateway
      context: api-gateway
      **jib: {}**
```
Once I added it, I was able to see the debug console triggered and debugging started working just fine.

Thanks for quick response.

Awad
